coderedcms/models/page_models.py

```diff
@@ -122,6 +118,28 @@ def __init__(cls, name, bases, dct):
         if not cls._meta.abstract:
             CODERED_PAGE_MODELS.append(cls)
 
+    @property
+    def search_db_include(self):
```

Comment:
Do deprecation notices work if you continue to specify search_db_include on a model, the old way? Or would your attribute end up overriding this pre-defined one?

And the flip side, if you call `page.search_db_include`, did you confirm this warning actually fires?